src/main/java/science/icebreaker/data/request/EditWikiPageRequest.java

```diff
@@ -0,0 +1,46 @@
+package science.icebreaker.data.request;
+
+import javax.validation.constraints.NotNull;
+
+public class EditWikiPageRequest {
+
+    @NotNull
+    private String title;
+
+    @NotNull
+    private String description;
+
+    @NotNull
+    private String references;
+
```

Comment:
Keywords are missing but since they are a little more complicated, I really don't mind that we leave it out for now.